src/test/java/runners/BstackRunner.java

```diff
@@ -65,21 +30,14 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
         List<TestTemplateInvocationContext> desiredCapsInvocationContexts = new ArrayList<>();
 
         try {
+            JSONParser parser = new JSONParser();
+            mainConfig = (JSONObject) parser
+                        .parse(new FileReader("src/test/resources/conf/" + System.getProperty("config")));
             platformConfig = (JSONArray) mainConfig.get("environments");
-            commonCapsConfig = (Map<String, Object>) mainConfig.get("capabilities");
+            wss = (String) mainConfig.get("wss");
 
             for (int i = 0; i < platformConfig.size(); i++) {
                 JSONObject platform = (JSONObject) platformConfig.get(i);
-                platform.putAll(commonCapsConfig);
-                platform.put("browserstack.username", username);
```

Comment:
Fetch username & accesskey from env var or yml file so if user doesn't have IDE plugin install and runs via IDE it at least trigger a runs on BS.